package.json

```diff
@@ -64,7 +64,7 @@
     "@actions/core": "^1.10.1",
     "@actions/exec": "^1.1.1",
     "@actions/io": "^1.1.3",
-    "@microsoft/powerplatform-cli-wrapper": "^0.1.120",
+    "@microsoft/powerplatform-cli-wrapper": "^0.1.121",
```

Comment:
Unlike PPBT and our other GH Repos, Github Actions here is unusual in that the repo is both the _source_ and the "build artifact".   
After updating CLI Wrapper, we also need to run `npm run update-dist` per the [readme](https://github.com/microsoft/powerplatform-actions/blob/main/README.md#refreshing-actions-in-dist-folder) to transpile and bundle to JS, and check in those generated files.